php/example_code/sns/README.md

```diff
@@ -0,0 +1,35 @@
+# Amazon Simple Notification Service (SNS) AWS SDK for PHP - Version 3 code examples
```

Comment:
Amazon SNS with AWS SDK for PHP Version 3 code examples
(short name, no parentheses in titles, also remove the dash from PHP Version)

Reply:
Done